### PR TITLE
Feature dbet 783 event subscriptions

### PR DESCRIPTION
**Pull request by molekilla**

### CODE REVIEW

File: src/Components/Wallet/Wallet.jsx [hunk 1]
```diff
@@ -603,13 +615,11 @@ class Wallet extends Component {
         let V2TokenBalance = this.state.balances.newToken.amount
         const vetAddress = keyHandler.getPubAddress()
         // QA Values, needs to be removed for alpha, beta or production
```
Comment (zubietaroberto) [inline src/Components/Wallet/Wallet.jsx]:
```suggestion
```

File: src/Components/Wallet/Wallet.jsx [hunk 2]
```diff
@@ -603,13 +615,11 @@ class Wallet extends Component {
...
-//        V1TokenBalance = 18080000000000000
-//        V2TokenBalance = 18080000000000000
+        // V1TokenBalance = 18080000000000000
```
Comment (zubietaroberto) [inline src/Components/Wallet/Wallet.jsx]:
```suggestion
```

File: src/Components/Wallet/Wallet.jsx [hunk 3]
```diff
@@ -603,13 +615,11 @@ class Wallet extends Component {
...
+        // V2TokenBalance = 18080000000000000
```
Comment (zubietaroberto) [inline src/Components/Wallet/Wallet.jsx]:
```suggestion
```

File: src/Components/Web3/Contracts/DBETToVETDepositContract.ts [hunk 4]
```diff
@@ -1,44 +1,41 @@
...
         super(web3)
-        this.listener = null
         this.contract = new web3.eth.Contract(
             Contract_DBETToVETDeposit.abi,
             Config.depositAddress
         )
+
+        const SENDER_CONTRACT_ADDRESS =
+            Config.env === 'production'
+                ? contracts.DBETVETToken.address['0xc7']
```
Comment (decent-dev) [inline src/Components/Web3/Contracts/DBETToVETDepositContract.ts]: We could probably add chainTag to Config and call `Config.chainTag` here instead

**Pull request merged**